python/pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [project]
 name = "pylance"
-dependencies = ["pyarrow>=12", "numpy>=1.22,<2"]
+dependencies = ["pyarrow>=12", "numpy>=1.22,<2", "cuvs-cu12", "pylibraft-cu12"]
```

Comment:
lets not put this into required dependencies?

Reply:
Done